.github/pull_request_template.md

```diff
@@ -1,35 +1,29 @@
-# Description of PR
-
-### Summary of changes and related issue
+## Summary of changes and related issue
 [What's changed in this PR?]
 
-### Relevant motivation and context
+## Relevant motivation and context
 [Why did you change this and what applicable context is needed to understand why this change is needed?]
 
-### Type of change
+## How to test 
+[How should reviewer's test the changes?] 
+
+## Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
+- [ ] None of the above  
```

Comment:
Out of curiosity what would fit in this category?

Reply:
haha! something like this PR :) not a new feature, bug fix, or a breaking change

Reply:
Ah gotcha, thanks. I can also see package version updates or something of the like here too